File: app/src/main/java/com/suishi/live/app/ui/fragment/PushFragment.java
package com.suishi.live.app.ui.fragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.suishi.live.app.R;

/**
 * 推流
 */
public class PushFragment extends Fragment {

    private static final String ARG_PARAM1 = "param1";
    private static final String ARG_PARAM2 = "param2";

    private String mParam1;
    private String mParam2;

    private static PushFragment fragment=null;


    public PushFragment() {
        // Required empty public constructor
    }

    /**
     */
    public static PushFragment newInstance() {
        if(fragment==null) {
            fragment = new PushFragment();
            Bundle args = new Bundle();
            fragment.setArguments(args);
        }
        return fragment;
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (getArguments() != null) {
            mParam1 = getArguments().getString(ARG_PARAM1);
            mParam2 = getArguments().getString(ARG_PARAM2);
        }
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_push, container, false);
    }


    @Override
    public void onDetach() {
        super.onDetach();
    }

}

File: app/src/main/java/com/suishi/live/app/ui/fragment/PushFragment.kt
package com.suishi.live.app.ui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.suishi.live.app.R

/**
 * 推流
 */
class PushFragment : Fragment() {
    private var mParam1: String? = null
    private var mParam2: String? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (arguments != null) {
            mParam1 = arguments!!.getString(ARG_PARAM1)
            mParam2 = arguments!!.getString(ARG_PARAM2)
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_push, container, false)
    }

    override fun onDetach() {
        super.onDetach()
    }

    companion object {
        private const val ARG_PARAM1 = "param1"
        private const val ARG_PARAM2 = "param2"
        private var fragment: PushFragment? = null

        /**
         */
        fun newInstance(): PushFragment? {
            if (fragment == null) {
                fragment = PushFragment()
                val args = Bundle()
                fragment!!.arguments = args
            }
            return fragment
        }
    }
}